src/data-display/dynamic/dynamic-layout/templates/raw-table/index.vue

```diff
@@ -74,6 +74,12 @@ export default defineComponent<RawTableDynamicLayoutProps>({
             fields: computed(() => {
                 if (state.rootData[0]) {
                     const firstItem = state.rootData[0];
+                    if (props.options?.headers) {
```

Comment:
suggestion: If there's no need to check `[]` case, ignore this.

```suggestion
                    if (props.options?.headers?.length) {
```